core/src/main/scala/org/apache/spark/SecurityManager.scala

```diff
@@ -282,7 +282,9 @@ private[spark] class SecurityManager(sparkConf: SparkConf)
       }: TrustManager
     })
 
-    val sslContext = SSLContext.getInstance(fileServerSSLOptions.protocol.getOrElse("Default"))
+    require(fileServerSSLOptions.protocol != None,
```

Comment:
It's fine though you could also write `.isDefined` here, which might be a millimeter more canonical. Some would use `getOrElse` to throw an exception, but `require` seems a little tidier for this purpose. Of course the change's substance looks fine.